messages/en.json

```diff
@@ -4,6 +4,9 @@
   "store/shipping-calculator.edit": "edit",
   "store/shipping-calculator.estimate": "estimate",
   "store/shipping-calculator.viewDeliveryOptions": "View delivery options",
+  "store/shipping-calculator.allItemsUnavailable": "All products can't be delivered to this address.",
+  "store/shipping-calculator.unavailableItems": "{numberOfUnavailableItems, plural, one {# product} other {# products}} that can't be delivered to this address are highlighted.",
+  "store/shipping-calculator.optionsForRemainingProducts": "{numberOfOptions, plural, one {Option} other {Options}} for remaining {numberOfItems, plural, one {product} other {products}}",
```

Comment:
You only treated plural for this message in `en`. Is this intentional?

Reply:
It's not. thanks!

Reply:
First message should be: "All the products can't be delivered to this address."
Sorry for the change 😉